applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,10 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_specific_name:
+            user_specific_bt_name = "NRF5340_AUDIO_" + getpass.getuser().upper()
```

Comment:
As stated in my other comment this makes sense for us here, developing inhouse, but it might not make sense for anyone else, as this will overwrite their specific BT_DEVICE_NAME with NRF5340_AUDIO_....